### ISSUE TITLE
Dynamic update of analysis page

### ISSUE THREAD

**Issue by aditirao7** [rottingresearch/rottingresearch#169]
It seems like the analysis page lost the functionality of updating dynamically as and when the link results are available? I'm pretty sure that it used to update dynamically before. Is this the desired behaviour @marshalmiller ?

**Comment by marshalmiller** [comment]
@aditirao7 Was this in the version from last week? Or has this been for a bit? I still believe the functionality is there. The issue is that we are now using a broker to accomplish tasks. The tasks are sent off to Celery and returned when finished. Without this, many runs were timing out, and there was little ability to run multiple sessions simultaneously. So, we went to the broker method. It has sped up a bit, but this dynamic updating might have been a casualty. If this is new to this week, I can go back and see where a conflict might have occurred.

**Comment by aditirao7** [comment]
@marshalmiller I'm not sure about the timeline of the change but ever since I started contributing again it was already like this. Currently it seems like it waits for all the celery tasks to finish and then shows the final results?

**Comment by marshalmiller** [comment]
Yes. I believe that is correct. I would love for that not to be the case, but I'm not sure of a way to accomplish both.

**Comment by marshalmiller** [comment]
@aditirao7 I appears like since you made your design changes for the PDF, the site does not return anything. I can look through the celery logs and everything is fine there. So it is an issue with the analysis page and you're the only one I can see has made changes to that recently. Any Ideas what is going on?

**Comment by aditirao7** [comment]
@marshalmiller It was working fine on local up till my changes, after pulling the new changes it stopped working. Also looks like my changes for the new pdf report format also got reverted.

**Comment by aditirao7** [comment]
**UPDATE**: Reverted to my latest [commit](https://github.com/rottingresearch/rottingresearch/commit/68475a62d1888f335f58c3280ae9f7cb9d4aa4cc) and it worked perfectly, after that I added the next [commit](https://github.com/rottingresearch/rottingresearch/commit/60d4c13f99367a6863e497ba94c83bd700f17ac4) and it stopped working. So that change somehow broke the analysis.

**Comment by marshalmiller** [comment]
@aditirao7 I'm sorry if I messed it up. Thank you for resolving it all, though. I really appreciate it. The design is just changing. Final touches should be good soon. Idk why the PDF report was touched though. I can look into that.

**Comment by aditirao7** [comment]
@marshalmiller No worries! Yep the PDF report part was removed maybe because of rebasing issues
